frontend/packages/kubevirt-plugin/src/k8s/patches/vm/vm-cpu-patches.ts

```diff
@@ -0,0 +1,37 @@
+import { PatchBuilder, PatchOperation } from '@console/shared/src/k8s';
+import { VMLikeEntityKind } from '../../../types';
+import { getVMLikePatches } from '../vm-template';
+import {
+  getCPU,
+  getResourcesRequestsCPUCount,
+  getResourcesLimitsCPUCount,
+  asVM,
+} from '../../../selectors/vm';
+
+export const getDedicatedCpuPatch = (
+  vmLikeEntity: VMLikeEntityKind,
+  dedicatedCpuPlacement: boolean,
+) => {
```

Comment:
its better to add return type to have clear function contract

```suggestion
): Patch[] => {
```